src/app/components/SelectNode/LightningApp.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { browser } from 'webextension-polyfill-ts';
+import { Alert, Button, message } from 'antd';
+import { urlWithoutPort } from 'utils/formatters';
+import './LightningApp.less';
+import { checkNode } from 'modules/node/actions';
+import { connect } from 'react-redux';
+import { AppState } from 'store/reducers';
+import TitleTemplate from 'components/SelectNode/TitleTemplate';
+import { DEFAULT_NODE_URLS } from 'utils/constants';
+import Spin from 'antd/lib/spin';
+
+interface StateProps {
+  isNodeChecked: AppState['node']['isNodeChecked'];
+  isCheckingNode: AppState['node']['isCheckingNode'];
+  checkNodeError: AppState['node']['checkNodeError'];
+}
+
+interface DispatchProps {
+  checkNode: typeof checkNode;
+}
+
+interface OwnProps {
+  onComplete(): void;
+}
+
+type Props = StateProps & DispatchProps & OwnProps;
+
+interface State {
+  error: string | undefined;
+}
+
+class LightningApp extends React.Component<Props, State> {
+
+  componentDidUpdate() {
+    if (this.props.isNodeChecked) {
+      this.props.onComplete();
+    }
+  }
+
+  componentDidMount() {
+    this.checkNode();
+  }
+
+  render() {
+    const {checkNodeError, isCheckingNode} = this.props;
+
+    return (
+      <div>
+        <TitleTemplate title={'Connecting to Lightning App...'}/>
+        {isCheckingNode && <Spin/> }
+        {checkNodeError &&
+        <Alert
+            type="error"
+            message="Failed to connect to node"
+            description={<>
+              <p>Request failed with the message
+                "{checkNodeError.message}"</p>
+              <p>
+                If you're sure you've setup your node correctly, try{' '}
+                <a href={`${DEFAULT_NODE_URLS.LIGHTNING_APP}/v1/getinfo`}
+                   target="_blank">
+                  clicking this link
+                </a>{' '}
+                and making sure it loads correctly. If there are SSL errors,
+                click "advanced" and proceed to accept the certificate.
+              </p>
+            </>}
+            showIcon
+            closable
+        />
```

Comment:
Might be good to componentize this since it's copy / pasted from InputAddress, and will likely be reused for a Zap onboarding component.